instructure/canvas-ios
# NetworkAvailabilityService component

refs: MBL-16602
affects: Student, Teacher, Parent
release note: none
test plan: 
1. Create an NWAvailabilityServiceLive instance in StudentAppDelegate
2. Call the startMonitoring function
3. Check the instance's status

## Checklist
